runtime/java/build.xml

```diff
@@ -28,7 +28,7 @@
     <javac debug='on' classpathref='compile.classpath' srcdir='${src}:${grammars}' destdir='${bin}' includeantruntime='false'>
       <include name='common/**/*.java' />
       <include name='core/*.java' />
-      <include name='monto/*.java' />
+      <!-- <include name='monto/*.java' /> -->
```

Comment:
We should maybe just remove this code (in a separate PR) rather than just letting it sit around and not be built.  @ericvanwyk?